simcon_project/templates/student_settings_view.html

```diff
@@ -19,7 +19,7 @@
 
 
     <title>
-    {% block title %}Simulated Conversations | Settings{% endblock %}
+    {% block title %}SimCon| Settings{% endblock %}
     </title>
       <script src="{% static 'js/jquery.bootstrap.modal.forms.js' %}"></script>
 
```

Comment:
The h1 tag here should be removed to be similar to researcher settings.

Reply:
ill delete it!